pkg/document/checkpoint/checkpoint_test.go

```diff
@@ -0,0 +1,37 @@
+package checkpoint_test
+
+import (
+	"github.com/stretchr/testify/assert"
+	"github.com/yorkie-team/yorkie/pkg/document/checkpoint"
+	"testing"
+)
+
+func TestCheckPoint(t *testing.T) {
+	t.Run("check point test", func(t *testing.T) {
+		cp := checkpoint.New(0, 0)
+		assert.Equal(t, false, cp.Equals(checkpoint.New(1, 1)))
+		assert.Equal(t, true, cp.Equals(checkpoint.New(0, 0)))
+		assert.Equal(t, false, cp.Equals(checkpoint.New(1, 0)))
+		assert.Equal(t, false, cp.Equals(checkpoint.New(0, 1)))
+		assert.Equal(t, false, cp.Equals(checkpoint.New(1, 1)))
```

Comment:
We can simply express boolean assertions using `assert.True` and `assert.False`.

e.g.) `assert.True(t, cp.Equals(checkpoint.New(1, 1)))`